test/libsolidity/analysis/FunctionCallGraph.cpp

```diff
@@ -1114,7 +1114,6 @@ BOOST_AUTO_TEST_CASE(interfaces_and_abstract_contracts)
 	unique_ptr<CompilerStack> compilerStack = parseAndAnalyzeContracts(R"(
 		interface I {
 			event Ev(uint);
-			modifier m() virtual;
```

Comment:
Damn, we even had a test for it?!

Reply:
Yeah, though when working on the `FunctionCallGraph` I just tried everything that compiled so it's not like it was by design.